### PR TITLE
Added path param to the install methods

### PR DESCRIPTION
**Pull request by nicolas-meilan**
Added the option of select the path to install the webDrivers

### CODE REVIEW

**Comment by SergeyPirogov** [comment]
Hello, thank you for the PR. As I understand this will allow saving driver in a folder that is different from `${USER_HOME}/.wdm/`. Am I Right? Why is This handy?

**Comment by nicolas-meilan** [comment]
Hello. You're welcome.
Yes, you are right. This is handy because, for example, I am creating a library to help me generate test projects with a particular structure and I am implementing the tool you have created to download the webDrivers that I do not have installed. But I install the drivers in the root project path, and later, when I can write the selenium tests in the project, I can use the webDriver.Chrome () method without typing the path of the webDriver. If I install webDrivers in {USER_HOME} /. Wdm / I will always have to use webDriver.Chrome ({USER_HOME} / .Wdm /) when writing tests.

**Comment by SergeyPirogov** [comment]
But DriverManager will do the job automatically. You just need to write
```
webdriver.Chrome(ChromeDriverManager().setup())
```
and that's all. I will automatically update driver version (if needed) and set the path to this driver

**Comment by nicolas-meilan** [comment]
It is okay, but i think that could be handy to have the option of select the path of the installation, because could be necessary use different versions of a webDriver in differents projects, for compatibilities problems and other problems that could happen. Installing the webDriver always in the same path, when a project is update, all projects are altered. it would a option if someone had have this problem. for this, I install the webdrivers in the root path of the projects.

**Comment by SergeyPirogov** [comment]
Ok. could you add some tests for this?

**Comment by SergeyPirogov** [comment]
@nicolas-meilan There are some failed tests here, please check

**Comment by nicolas-meilan** [comment]
Some tests failed because exists a problem with the max number of downloads, I'll try to solve it later

File: tests/test_edge_driver.py [hunk 1]
```diff
@@ -9,20 +8,33 @@
...
 
 @pytest.mark.skipif(sys.platform != 'win32',
                     reason="run only on windows")
-def test_edge_manager_with_selenium_cache():
-    driver_path = EdgeDriverManager().install()
+@pytest.mark.parametrize('path', PATH)
+@pytest.mark.parametrize('with_path', [True,
+                                       False])
+def test_edge_manager_with_selenium_cache(path, with_path):
+    if with_path:
```
Comment (SergeyPirogov) [inline tests/test_edge_driver.py]: I don't like this implementation of tests. There should not be any if statements

**Comment by nicolas-meilan** [comment]
I finished the tests, sorry for spending long time in this. I removed the ifs that you showed me

**Comment by SergeyPirogov** [comment]
Ok, good. Looks good

**Pull request merged**